core/src/main/java/org/axonframework/eventhandling/scheduling/java/SimpleEventScheduler.java

```diff
@@ -150,11 +154,86 @@ public void run() {
             EventMessage<?> eventMessage;
             if (event instanceof EventMessage) {
                 eventMessage = new GenericEventMessage<>(((EventMessage) event).getPayload(),
-                                                               ((EventMessage) event).getMetaData());
+                                                         ((EventMessage) event).getMetaData());
             } else {
                 eventMessage = new GenericEventMessage<>(event, MetaData.emptyInstance());
             }
             return eventMessage;
         }
     }
+
+    /**
+     * Builder class to instantiate a {@link SimpleEventScheduler}.
+     * <p>
+     * The {@link TransactionManager} is defaulted to a {@link NoTransactionManager}. The
+     * {@link ScheduledExecutorService} and {@link EventBus} are a <b>hard requirements</b> and as such should be
+     * provided.
+     */
+    public static class Builder {
+
+        private ScheduledExecutorService scheduledExecutorService;
+        private EventBus eventBus;
+        private TransactionManager transactionManager = NoTransactionManager.INSTANCE;
+
+        /**
+         * Sets the {@link EventBus} used to publish events on once the schedule has been met.
```

Comment:
`on`?

Reply:
I'll clarify it by changing it to `on to,`.